### PR TITLE
ru: Remove `{{geckoRelease}}` macro

### PR DESCRIPTION
**Pull request by queengooborg**
This PR removes the `{{geckoRelease}}` macro from Russian documents.  Regex used: `\s\{\{\s?geckoRelease\s?\(['"]?[\d.]+['"]?\)\s?\}\}` -> [Blank]

Note: some manual changes were needed due to original misuses of the macro.

### CODE REVIEW

File: files/ru/web/api/settimeout/index.md [hunk 1]
```diff
@@ -241,7 +241,7 @@ String literals are evaluated in the global context, so local symbols in the con
 
 [Historically](http://code.google.com/p/chromium/issues/detail?id=792#c10) browsers implement `setTimeout()` "clamping": successive `setTimeout()` calls with `delay` smaller than the "minimum delay" limit are forced to use at least the minimum delay. The minimum delay, `DOM_MIN_TIMEOUT_VALUE`, is 4 ms (stored in a preference in Firefox: `dom.min_timeout_value`), with a `DOM_CLAMP_TIMEOUT_NESTING_LEVEL` of 5ms.
 
-In fact, 4ms is [specified by the HTML5 spec](http://www.whatwg.org/specs/web-apps/current-work/multipage/timers.html#timers) and is consistent across browsers released in 2010 and onward. Prior to {{ geckoRelease("5.0") }}, the minimum timeout value for nested timeouts was 10 ms.
+In fact, 4ms is [specified by the HTML5 spec](http://www.whatwg.org/specs/web-apps/current-work/multipage/timers.html#timers) and is consistent across browsers released in 2010 and onward. Prior to, the minimum timeout value for nested timeouts was 10 ms.
```
Comment (yin1999) [inline files/ru/web/api/settimeout/index.md:244]:
```suggestion
In fact, 4ms is [specified by the HTML5 spec](http://www.whatwg.org/specs/web-apps/current-work/multipage/timers.html#timers) and is consistent across browsers released in 2010 and onward. Prior to Gecko 5.0, the minimum timeout value for nested timeouts was 10 ms.
```

File: files/ru/web/api/settimeout/index.md [hunk 2]
```diff
@@ -251,7 +251,7 @@ Browsers including Internet Explorer, Chrome, Safari, and Firefox store the dela
 
 #### Неактивные вкладки
 
-In {{ geckoRelease("5.0") }} and Chrome 11, timeouts are clamped to firing no more often than once per second (1000ms) in inactive tabs; see {{ bug(633421) }} for more information about this in Mozilla or [crbug.com/66078](http://crbug.com/66078) for details about this in Chrome.
+In and Chrome 11, timeouts are clamped to firing no more often than once per second (1000ms) in inactive tabs; see {{ bug(633421) }} for more information about this in Mozilla or [crbug.com/66078](http://crbug.com/66078) for details about this in Chrome.
```
Comment (yin1999) [inline files/ru/web/api/settimeout/index.md:254]:
```suggestion
In Gecko 5.0 and Chrome 11, timeouts are clamped to firing no more often than once per second (1000ms) in inactive tabs; see {{ bug(633421) }} for more information about this in Mozilla or [crbug.com/66078](http://crbug.com/66078) for details about this in Chrome.
```

**Review by yin1999 — APPROVED** [review]
It looks good to me, just two nits

**Review by sashasushko — APPROVED** [review]

**Comment by sashasushko** [comment]
@queengooborg спасибо 🙏

**Pull request merged**